### PR TITLE
Add test to spring-boot-sample-quartz

### PR DESCRIPTION
**Pull request by eddumelendez**
See gh-11613

### CODE REVIEW

File: spring-boot-samples/spring-boot-sample-quartz/pom.xml [hunk 1]
```diff
@@ -19,6 +19,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-quartz</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
```
Comment (snicoll) [inline spring-boot-samples/spring-boot-sample-quartz/pom.xml]: This is missing a `test`scope.

File: spring-boot-samples/spring-boot-sample-quartz/src/test/java/sample/quartz/SampleQuartzApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * @author Eddú Meléndez
+ */
+public class SampleQuartzApplicationTests {
+
+	@Rule
+	public OutputCapture outputCapture = new OutputCapture();
+
+	@Test
+	public void test() {
+		SampleQuartzApplication.main(new String[0]);
```
Comment (snicoll) [inline spring-boot-samples/spring-boot-sample-quartz/src/test/java/sample/quartz/SampleQuartzApplicationTests.java]: Can you please get the context back there and close it at the end? It is probably useless now but it's better to make sure we actually trigger a shutdown.

File: spring-boot-samples/spring-boot-sample-quartz/pom.xml (hunk 1 above)
Comment (eddumelendez) [reply spring-boot-samples/spring-boot-sample-quartz/pom.xml]: 😱

File: spring-boot-samples/spring-boot-sample-quartz/src/test/java/sample/quartz/SampleQuartzApplicationTests.java (hunk 2 above)
Comment (eddumelendez) [reply spring-boot-samples/spring-boot-sample-quartz/src/test/java/sample/quartz/SampleQuartzApplicationTests.java]: should I use `ApplicationContextRunner`?

Comment (snicoll) [reply spring-boot-samples/spring-boot-sample-quartz/src/test/java/sample/quartz/SampleQuartzApplicationTests.java]: I  don't think so. Running the app is ok, we have other samples that do that.

File: spring-boot-samples/spring-boot-sample-quartz/pom.xml (hunk 1 above)
Comment (eddumelendez) [reply spring-boot-samples/spring-boot-sample-quartz/pom.xml]: fixed

**Pull request closed without merging**